src/stream/src/task/barrier_manager/managed_state.rs

```diff
@@ -416,32 +424,27 @@ impl PartialGraphManagedBarrierState {
     }
 }
 
-pub(super) struct ManagedBarrierState {
+pub(crate) struct ManagedBarrierState {
     pub(super) actor_states: HashMap<ActorId, InflightActorState>,
```

Comment:
May I know what "inflight" in `InflightActor` is for?

Reply:
It has no special meaning. I guess the name was following the `InflightActorInfo` previously in the meta node global barrier manager.